tests/apt_diff_expected.json

```diff
@@ -7,76 +7,76 @@
       "Packages1": {
         "dh-python": {
           "Version": "2.20170125",
-          "Size": "402"
+          "Size": 411648
         },
         "libmpdec2": {
           "Version": "2.4.2-1",
-          "Size": "254"
+          "Size": 260096
         },
         "libpython3-stdlib": {
           "Version": "3.5.3-1",
-          "Size": "36"
+          "Size": 36864
         },
         "libpython3.5-minimal": {
           "Version": "3.5.3-1",
-          "Size": "3747"
+          "Size": 3836928
         },
         "libpython3.5-stdlib": {
           "Version": "3.5.3-1",
-          "Size": "9896"
+          "Size": 10133504
```

Comment:
just seems like these numbers are pretty different. I trust the new method more though.